### ISSUE TITLE
Make use of NYX preRelease / draft for GitHub and Docker

### ISSUE THREAD

**Issue by clemens-tolboom** [molgenis/molgenis-service-armadillo#586]
We want to distinguish between release and pre release

- in particular docker images
- on GitHub

Reading the docs for release types

- https://mooltiverse.github.io/nyx/guide/user/configuration-reference/release-types/

we have

- [publish](https://mooltiverse.github.io/nyx/guide/user/configuration-reference/release-types/#publish)
- [publishDraft](https://mooltiverse.github.io/nyx/guide/user/configuration-reference/release-types/#publish-draft)
- [publishPreRelease](https://mooltiverse.github.io/nyx/guide/user/configuration-reference/release-types/#publish-pre-release)

The last 2 needs `publish = true` in 

https://github.com/molgenis/molgenis-service-armadillo/blob/4fdbe94a356ad379cb2d44f605348b0485d944d6/settings.gradle#L13-L17

but we need this use these as well in 

https://github.com/molgenis/molgenis-service-armadillo/blob/4fdbe94a356ad379cb2d44f605348b0485d944d6/build.gradle#L127-L134

Currently snapshots are PRs only

**Comment by clemens-tolboom** [comment]
### GitHub

Would it be enough for GitHub settings?

```Gradle
mainline { 
     matchBranches = "master" 
     publish = "true"
     publishPreRelease = "true"  <========================
     description = "{{#fileContent}}CHANGELOG.md{{/fileContent}}" 
     publicationServices = ["github"] 
```

### Docker

What are snapshot candidates? And should we use NYX logic for that instead?

**Comment by clemens-tolboom** [comment]
We have updated nyx in 31da0dca4578843a45bbe4bd922b6fbc151112bc

**Comment by clemens-tolboom** [comment]
Fixed through #604
